package-lock.json

```diff
@@ -3580,4 +3580,4 @@
       "dev": true
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
ideally this file should not be in commit.

Reply:
How do I remove it from the commit? I reverted the changes I made.

Reply:
```
git checkout main -- package-lock.json
```

then check and commit the change, push. That should make it exactly same as main, and it should disappear from the PR.